Sources/GRPCProtobuf/ContiguousBytesAdapter.swift

```diff
@@ -63,3 +63,6 @@ struct ContiguousBytesAdapter<
     try self.bytes.withUnsafeMutableBytes(body)
   }
 }
+
+@available(gRPCSwiftProtobuf 2.0, *)
```

Comment:
This will be available in 2.1:

```suggestion
@available(gRPCSwiftProtobuf 2.1, *)
```